content/_kss/info/identity-lockup.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"header":"Lockup","description":"<p>A lockup with the Stanford wordmark logo and department/unit name. It is used in the header of a website and links to the homepage of the department/unit. When building a department lockup, besides the Stanford wordmark, the most obvious element should be the most granular relevant department.</p>\n<p>.su-lockup--option-a - Option A\n.su-lockup--option-b - Option B\n.su-lockup--option-c - Option C\n.su-lockup--option-d - Option D\n.su-lockup--option-e - Option E\n.su-lockup--option-f - Option F\n.su-lockup--option-g - Option G\n.su-lockup--option-h - Option H\n.su-lockup--option-i - Option I\n.su-lockup--option-j - Option J\n.su-lockup--option-k - Option K\n.su-lockup--option-l - Option L\n.su-lockup--option-m - Option M\n.su-lockup--option-n - Option N\n.su-lockup--option-o - Option O\n.su-lockup--option-p - Option P\n.su-lockup--option-q - Option Q\n.su-lockup--option-r - Option R\n.su-lockup--option-s - Option S\n.su-lockup--option-t - Option T</p>\n<p>@deprecated Options C, G, J, &amp; Q will be removed in Version 7</p>\n","deprecated":false,"experimental":false,"reference":"Identity.Lockup","referenceNumber":"2.4","referenceURI":"identity-lockup","weight":0,"colors":[],"markup":"templates/components/lockup/lockup.twig","source":{"filename":"lockup/_lockup.scss","line":3},"depth":2,"modifiers":[],"parameters":[],"source_twig":"templates/components/lockup/lockup.twig"}
```

Comment:
Ohh one question - any idea how this happened? Looks like we only have the default lockup on this page on preview, where on live there are multiple variants 🤔  Maybe Dena would know about this?
https://deploy-preview-89--elegant-poitras-87214a.netlify.app/component/identity-lockup/

Reply:
@sherakama fix resolved this issue! Thanks again, Shea! 👍